File: Assignment/Module-1(fundamentals/p4.cpp
#include<stdio.h>
main()
{
	double radius,area;
	const double pi=3.14;
	printf("Enter the radius of the circle:");
	scanf("%lf",&radius);
	if(radius>=0)
	{
		area = pi*radius*radius;
		printf("The area of the circle with radius %.2lf is %.22lf \n",radius,area);
	}
	else
	{
		printf("radius cannot be negative.\n");
	}
}

File: Assignment/flutter dart assignment module-1/p4.dart
import 'dart:io';

void main() {

  stdout.write("Enter the radius of the circle: ");
  double radius = double.parse(stdin.readLineSync()!);

  double area = 3.14159 * radius * radius;

  print("The area of the circle with radius $radius is $area square units.");
}
